<!DOCTYPE html>
<html lang="en"><head>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-124828585-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-124828585-1');
</script>



<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <title>Yi Zhang, PhD candidate of UIUC, compbio</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Yi">
    <meta name="author" content="Yi">
	<meta name="keywords" content="Yi Zhang,  computational biology, bioinformatics, cancer genetics, UIUC, USTC, bioengineering, big data, biomodical, machine learning, statistical learning">
    <!-- Le styles -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <link href="materials/additional.css" rel="stylesheet">
    <style>
  body {
        padding-top: 60px; /* 60px to make the container go all the way to the bottom of the topbar */
      }
    </style>
  </head>

  <body>

  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
    <a class="navbar-brand" href="#HOME-anchor">Yi's Homepage</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
      <div class="navbar-nav">
        <a class="nav-item nav-link" href="#HOME-anchor">About me</a>
        <a class="nav-item nav-link" href="#Publications-anchor">Publications</a>
        <a class="nav-item nav-link" href="#Projects-anchor">Projects</a>
      </div>
    </div>
  </nav>


    <div class="container">

    <a class="anchor" id="HOME-anchor"></a>
		<section id="HOME">
			<div class="page-header">
                <h1>Yi Zhang &nbsp<small><font color="gray">  Ph.D. Candidate</font></small></h1>
			</div>

			<div class="row">

				<div class="col-3">
          <img src="materials/your_photo.png" alt="Responsive image" class="img-thumbnail">
	

                    <address>
                        <br>
                        <strong>Address:  </strong>
                        2132 Institute for Genomic Biology,
                        1206 West Gregory Drive,<br> Urbana, IL 61801, USA<br>
                        <strong>Email:  </strong>
						yzhan201 [AT] illinois [DOT] edu<br>
                        <!--[<a href="#">Github</a>]-->
                        [ <a href="https://scholar.google.com/citations?user=yTJVsxEAAAAJ&hl=en" target="_blank">Google Scholar</a> ]
                        [ <a href="materials/CV_Yi_Zhang_UIUC_2018.pdf" target="_blank">CV</a> ]<br>
                    </address>

				</div>

                <div class="col-9">
                    <p>I am a Ph.D. student at the <a href="https://bioengineering.illinois.edu" target="_blank">Department of Bioengineering</a> working in the <a href="http://song.igb.illinois.edu" target="_blank">Computational Biology Group</a> and the <a href="http://igb.illinois.edu/" target="_blank">Carl R. Woese Institute for Genomic Biology</a>, at <a href="http://illinois.edu/" target="_blank">University of Illinois at Urbana-Champaign</a>. I have been developing computational biology and statistical learning methods for analyzing and understanding large-scale biomedical data, with a focus on establishing links from human genetic variants to disease phenotypes like cancer. My research interests fall in genetics/epigenetics of diseases like cancers, developing statistical and machine learning methods for biomedical big data analysis, and so on. For fun, I enjoy hiking, music and wildlife photographing.</p>
                    <br>
                    <div class="page-header">
                        <p><h3> Education</h3></p>
                    </div>
                    <p><strong>Ph.D. Candidate</strong>, Aug. 2014 - May 2019 (expected)</p>
                
                    <ul>
                    <li>
                        Department of Bioengineering, University of Illinois at Urbana-Champaign         
                    </li>
                    <!--<li>
						<a href="http://song.igb.illinois.edu/" target="_blank">Computational Biology Group</a><br>
                        </li>
                        -->
                    <li>
                    Advisor: <a href="http://song.igb.illinois.edu/" target="_blank">Professor Jun S. Song</a>         
                    </li>
                    </ul>

                    <p><strong>Bachelor of Science, Biosciences</strong>, Sep. 2010 - Jul. 2014</p>
                   
                    
                    <ul>
                    <li>
                        Life Sciences Talent Program, University of Science and Technology of China         
                    </li>
                    <li>
                        Department of Life Sciences, University of Science and Technology of China         
                    </li>
                </ul>

				</div>
			
		</section>
		<br>

    <a class="anchor" id="Publications-anchor"></a>
		<section id="Publications">
			<div class="page-header">
				<h3>Publications</h3>
            </div>

			<div class="row">
                <div class="col-8">
                    <ol>
                        <li>
                        <strong><u>Y. Zhang*</u></strong>, M. Manjunath*, S. Zhang, D. Chasman, S. Roy, and J.S. Song. (*co-first authors). "<a href="http://cancerres.aacrjournals.org/content/78/7/1579.long" target="_blank">Integrative genomic analysis predicts causative cis-regulatory mechanisms of the breast cancer-associated genetic variant rs4415084.</a>" <strong><em>Cancer Research</em></strong>, 78(7), 1579-1591, (2018). 
                        </li>
                        <br>
                        <li>
                        <strong><u>Y. Zhang</u></strong>, M. Manjunath, Y. Kim, J. Heintz, and J.S. Song. "<a href="https://www.biorxiv.org/content/early/2018/07/08/319079" target="_blank">SequencEnG: an Interactive Knowledge Base of Sequencing Techniques.</a>" <em>Accepted, <strong>Bioinformatics</strong>,</em> (2018).<br>[<strong> <a href="http://education.knoweng.org/sequenceng/" target="_blank">SequencEnG page</a> </strong>]
                        </li>
                        <br>


                        <li>
                        M. Manjunath, <strong><u>Y. Zhang</strong></u>, S.H. Yeo, O. Sobh, N. Russell, C. Followell, C. Bushell, U. Ravaioli and J.S. Song. "<a href="https://peerj.com/articles/cs-155/" target="_blank" >ClusterEnG: an interactive educational web resource for clustering and visualizing high-dimensional data.</a>" <strong><em>PeerJ Computer Science</em></strong> 4, e155, (2018). [ <a href="http://education.knoweng.org/clustereng/" target="_blank">ClusterEnG page </a> ]
                        </li>
                    </ol>
                </div>
                <div class="col-2" >
                    <!-- altmetric sequenceng -->
                <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1158/0008-5472.CAN-17-3486"></div>
                    <!-- altmetric cancerres -->
                 <div class='altmetric-embed' data-badge-type='donut' data-doi="doi.org/10.1101/319079"></div>
                    <!-- altmetric clustereng -->
                 <div class='altmetric-embed' data-badge-type='donut' data-doi="doi.org/10.7717/peerj-cs.155"></div>
                </div>

			</div>

        </section>

		<section id="Poster Presentations">
			<div class="page-header">
				<h3>Poster Presentations</h3>
            </div>

			<div class="row">
                <div class="col-8">
                    <ol>
                        <li>
                        <strong><u>Y. Zhang</u></strong>, M. Manjunath, S. Zhang, D. Chasman, S. Roy, & J.S. Song. "Abstract 1220: Integrative genomic analysis discovers the causative regulatory mechanisms of a breast cancer-associated genetic variant." <em><strong>AACR Annual Meeting</strong>, 2018 Apr. 14-18, Chicago, IL</em>, (2018). 
                        </li>
                    </ol>
                    <br>
                    <ul>
                        <li>
                        <u>Y. Zhang</u>, M. Manjunath, & J.S. Song, "Identifying Causal Mechanisms of Germline Risk Variants in Breast Cancer."<em> Bioengineering Graduate Student Symposium, UIUC</em>, (2017).
                        </li>
                        <li>
                        <u>Y. Zhang</u>, P. Gad, & V.R. Edgerton, "Burst Detection in EMG of Stepping Spinal Cord Injured Rats." <em>UCLA Cross-disciplinary Scholars in Science and Technology (CSST) summer research No. 69, 2010 Sep., Los Angeles, CA, </em>(2013).
                        </li>
                    </ul>
                </div>

			</div>

            </section>

    <a class="anchor" id="Projects-anchor"></a>
		<section id="Projects">
			<div class="page-header">
				<h3>Projects</h3>
			</div>

           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/web_pics.001.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>Computational methods for interpretation of cancer-associated genetic variants.</strong> [ <a href="http://cancerres.aacrjournals.org/content/78/7/1579.long">Paper</a> ]
               <p>Previous genome-wide association studies (GWAS) have identified common genetic variants that modulate cancer susceptibility, with their causative mechanisms missing. We have developed computational methods for functional interpretation of variants associated with cancer, by combining analyses of expression quantitative trait loci (eQTL), a modified version of allele-specific expression (ASE) that systematically utilizes haplotype information, transcription factor (TF) binding preference, and epigenetic information. Databases/techniques useful in this study are: TCGA, ENCODE, GTEx, Roadmap genomics, TF motif databases, genotyping array, RNA-seq, ChIP-seq, ChIA-PET, Hi-C, <em>etc.</em>. Application of our integrative methods to a breast cancer susceptibility region shows that a SNP, linked to a GWAS SNP, modulates GATA3 binding activity and in turn regulates the gene MRPS30 and lncRNA RP11-53O19.1. Our computational framework provides an effective means to integrate GWAS results with high-throughput genomic and epigenomic data. <br>We are currently working on extending the analysis to other variants, as well as applying machine learning methods to reveal genotypic-phenotypic relationship in context of cancer.<br>
               </p>
           </div>
           </div>
           <br>

           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/web_pics.002.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>SequencEnG: an interactive knowledge base of NGS techniques.</strong> [ <a href="https://www.biorxiv.org/content/early/2018/07/08/319079">Paper</a> ] &nbsp;  [ <a href="education.knoweng.org/sequenceng">SequencEnG page</a> ]&nbsp;
               <p>Next-generation sequencing (NGS) techniques are revolutionizing biomedical research by providing powerful methods for generating genomic and epigenomic profiles. However, a neat and barrier-free learning resource is lacking. We have developed an interactive online educational resource called SequencEnG (acronym for <strong><u>Sequenc</u></strong>ing Techniques <strong><u>En</u></strong>gine for <strong><u>G</u></strong>enomics) to provide a tree-structured knowledge base of <strong>71</strong> different sequencing techniques and step-by-step NGS data analysis pipelines comparing popular tools. SequencEnG is part of the project KnowEnG (Knowledge Engine for Genomics). I wish I could have a resource like this when I first entered bioinformatics :D<br>
               Update 2018-08: SequencEnG has ~3500 users in 3 months!
               </p>
           </div>
           </div>
           <br>

           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/web_pics.003.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>ClusterEnG: clustering and visualizing high-dimensional data.</strong><br> [ <a href="https://peerj.com/articles/cs-155/">Paper</a> ] &nbsp;  [ <a href="education.knoweng.org/clustereng">ClusterEnG page</a> ]&nbsp;
            <p>We have developed ClusterEnG (acronym for <u>Cluster</u>ing <u>En</u>gine for <u>G</u>enomics) to provide a web resource for online clustering analysis and interactive visualizations. ClusterEnG also serves as a barrier-free learning online resource of clustering algorithms with a focus on biomedical data. I have created cool visuliazations of clustering and dimensional reduction results for better understanding of data structure in an intuitive manner. 
            <br>
            </p>
           </div>
           </div>
           <br>

		</section>



		<section id="TEACHING">
			<div class="page-header"><h3>Talks & Teaching</h3></div>
            <ul>
                <li>Invited Speaker, Bioinformatics Seminar, <em>UIUC ANSC590</em>, 2016, 2018</li>
                <li>Speaker, Bioengineering Graduate Student Seminar, <em>UIUC</em>, 2018</li>
                <li>Teaching Assistant, Biomedical Instrumentation Lab, <em>UIUC BIOE415</em>, 2014-2015</li>
            </ul>
    <br>
    </section>


		<section id="PROF">
        <div class="page-header"><h3>Professional Experience</h3></div>
        <ul>
            <li>
            NHGRI Short Course on NGS: Technology & Statistical Methods, <em>Birmingham, AL</em>, 2016.
            </li>
            <li>
            BD2K (Big Data to Knowledge) KnowEnG center, <em>UIUC</em>, 2015-2018.
            </li>
        </ul>
    <br>
		</section>

		<section id="Courses">
        <div class="page-header"><h3>Related Courses</h3></div>
        <div class="row">
        <div class="col-10">
            <table class="table table-bordered">
            <tbody>
            <tr>
                    <td>Mathematical Statistics</td>
                    <td>Machine Learning</td>
                    <td>Computational Cancer Biology</td>
                </tr>
            <tr>
                    <td>Stochastic Processes</td>
                    <td>Statistical Learning</td>
                    <td>Statistical Data Analysis in Physics</td>
            </tr>
            </tbody>
        </table>
        <br>
        <h5>Course Projects</h5>
        <em>Statistical Learning</em>: Prediction of Movie votes and campus shooting with SVM and gradient boosting models.<br>
        <em>Machine Learning</em>: Classification of NGS technique articles using natural language processing and SVM models.<br>
        <em>Stochastic Processes</em>: Segmentation of copy number using HMM based on genotype data in cancer cells.<br>
        <em>Statistical Data Analysis in Physics</em>: A cross-entropy method for graph clustering and Cheeger constant estimation.<br>
        </div>
        </div>
        <br>
        </section>

		<section id="AWARDS">
			<div class="page-header"><h3>Honors and Awards</h3></div>
      <ul>
        <table width="150%" align="center" border="0" cellpadding="20">
            <li><span class='title'>Best Poster Award, <em>Bioengineering Graduate Annual Symposium, UIUC</em>, 2017</li>
            <li><span class='title'>UCLA-CSST Summer Research Scholarship, <em>UCLA</em>, 2013</li>
            <li><span class='title'>Outstanding Undergraduate Student Honor, <em>USTC</em>, 2013</li>
            <li><span class='title'>Outstanding Student Scholarship, School of Life Sciences, <em>USTC</em>, 2013, 2014</li>
            <li><span class='title'>Aegon-Industrial Foundation Scholarship, <em>USTC</em>, 2012</li>
            <li><span class='title'>Panasonic Elite Scholarship, <em>USTC</em>, 2011</li>
         </table>
      </ul>
      </section>

      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>

      <!-- Footer ================================================== -->
            <footer class="page-footer font-small">
                <!-- Copyright -->
                <div class="footer-copyright text-center py-3" style="background-color:#FFFFFF">
                <font color='green'>Â© 2018 Copyright: <strong> Yi Zhang</strong> (updated Aug 2018)</font>
                </div>
                </footer>

</body></html>
