<!DOCTYPE html>
<html lang="en"><head>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-124828585-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-124828585-1');
</script>



<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <title>Yi Zhang, PhD compbio </title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Yi">
    <meta name="author" content="Yi">
	<meta name="keywords" content="Yi Zhang, computational biology, bioinformatics, cancer genetics, UIUC, Harvard University, Dana-Farber, big data, biomodical, machine learning">
    <!-- Le styles -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <link href="materials/additional.css" rel="stylesheet">
    <style>
  body {
        padding-top: 60px; /* 60px to make the container go all the way to the bottom of the topbar */
      }
    </style>
  </head>

  <body>



<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
    <a class="navbar-brand abs" href="#HOME-anchor">Yi Zhang's Homepage</a>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
        <span class="navbar-toggler-icon"></span>
    </button>
    <div class="navbar-collapse collapse" id="collapsingNavbar">
        <ul class="navbar-nav">
            <li class="nav-item">
                <a class="nav-link" href="#HOME-anchor">About me</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#Publications-anchor">Publications</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#Projects-anchor">Projects</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#Courses-anchor">Courses</a>
            </li>
        </ul>
        <ul class="navbar-nav ml-auto">
            <li class="nav-item">
                <a class="nav-link active" href="photos.html">Wildlife</a>
            </li>
        </ul>
    </div>
</nav>

    <div class="container">

    <a class="anchor" id="HOME-anchor"></a>
		<section id="HOME">
			<div class="page-header">
                <h1>Yi Zhang &nbsp<small><font color="gray">  Ph.D.</font></small></h1>
			</div>

			<div class="row">

				<div class="col-3">
          <img src="materials/your_photo.png" alt="Responsive image" class="img-thumbnail">
	

                    <address>
                        <br>
                        <strong>Address:  </strong>
                        450 Brookline Ave, <br> Boston 02215 USA<br>
                        <strong>Email:  </strong>
						yi_zhang2 [AT] dfci [DOT] harvard [DOT] edu<br>
                        [ <a href="https://scholar.google.com/citations?user=yTJVsxEAAAAJ&hl=en" target="_blank">Google Scholar</a> ]
                        [ <a href="materials/CV_Yi_Zhang_2022.pdf" target="_blank">CV</a> ]<br>
                        <div itemscope itemtype="https://schema.org/Person"><a itemprop="sameAs" content="https://orcid.org/0000-0002-7453-6188" href="https://orcid.org/0000-0002-7453-6188" target="orcid.widget" rel="me noopener noreferrer" style="vertical-align:top;"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon">https://orcid.org/0000-0002-7453-6188</a></div><br>
                        <!--[<a href="#">Github</a>]-->
                        
                        
                        
                    </address>

				</div>

                <div class="col-9">
                    <p>I am a Research Fellow at the <a href="https://ds.dfci.harvard.edu/" target="_blank">Department of Data Science, Dana-Farber Cancer Institute</a> and <a href="https://www.hsph.harvard.edu/" target="_blank">Harvard University T.H. Chan School of Public Health</a>. I joined <a href="https://liulab-dfci.github.io/" target="_blank">Dr. X. Shirley Liu</a>'s lab in July 2019 and also co-mentored by <a href="https://mylesbrownlab.dana-farber.org/" target="_blank">Dr. Myles Brown</a> and <a href="https://content.sph.harvard.edu/xlin/people.html" target="_blank"> Dr. Xihong Lin</a>. I did my PhD in Bioengineering in <a href="http://song.igb.illinois.edu" target="_blank">Dr. Jun S. Song's group</a> at <a href="http://illinois.edu/" target="_blank">University of Illinois at Urbana-Champaign</a> working for NIH Big Data to Knowledge (BD2K) U54 center. <br>
                        I have developed computational biology and statistical learning methods to decipher human genetic variants associated with cancer risk. I have also been developing machine learning methods for single-cell genomics data to understand cells in tumor. <br>
                        In general, my research interests fall in developing statistical and machine learning methods for biomedical data-driven discoveries, with an ultimate aim of facilitating personalized cancer therapies. <br>  
                    I also enjoy achievement that helps understand disease to identify therapy based on patient's data. Thus, I have also been collaborating with scientists including Dr. Catherine J. Wu in DFCI and Dr. David E. Fisher at MGH.<br>
                    For fun, I enjoy hiking, music and wildlife photographing.</p>
                    <br>
                    <div class="page-header">
                        <p><h3> Education</h3></p>
                    </div>
                    <p><strong>Ph.D. Bioengineering</strong>, Aug. 2014 - May 2019</p>
                
                    <ul>
                    <li>
                        Department of Bioengineering, University of Illinois at Urbana-Champaign         
                    </li>
                    
                    <li>
                    Committee: Dr. Jun S. Song, Dr. Saurabh Sinha, Dr. Dave S. Zhao, Dr. Pablo Perez-Pinera         
                    </li>
                    </ul>

                    <p><strong>Bachelor of Science, Biosciences</strong>, Sep. 2010 - Jul. 2014</p>
                   
                    
                    <ul>
                    <li>
                        Life Sciences Talent Program, University of Science and Technology of China         
                    </li>
                    <li>
                        Department of Life Sciences, University of Science and Technology of China         
                    </li>
                </ul>

				</div>
			
		</section>
		<br>

    <a class="anchor" id="Publications-anchor"></a>
		<section id="Publications">
			<div class="page-header">
				<h3>Publications</h3>
            </div>

			<div class="row">
                <div class="col-12">
                    <ol>
                        <li>
                        <strong>Zhang Y</strong>, Xiang G, Fan J, Wang C, Lynch A, Zeng Z, Zhang W, Kang J, Jiang A, Gu S, Wan C, Zhang B, Brown M#, Liu XS#, Meyer C#. "MetaTiME: Meta-components of the Tumor Microenvironment." <em>Manuscript In Submission</em>. 
                        </li>
                        <br>

                        <li>
                            Choi Y*, Erlich T*, Franque M, Rachmin I, Flesher J, Schiferle E, <strong>Zhang Y</strong>, Silva M, Jiang A, Dobry A, Lacher S, Freund O, Feder E, Cortez J, Ryu S,  Samuels Y, Zakka L, Azin M, Burd C, Shapless NN, Liu XS, Meyer C, Austen W, Bojovic B, Certrulo C, Mihm M, Hoon D, Demehri S, Hawryluk E, Fisher DE#. “Topical therapy for regression and melanoma prevention of congenital giant nevi.” <strong><em>Cell</em></strong> (2022) Accepted .
                        </li>
                        <br>

                        <li>
                            Qiu X*, Boufaied N*, Hallal T, Feit A, Polo A de, Luoma AM, Alahmadi W, Larocque J, Zadra G, Xie Y, Gu S, Tang Q, <strong>Zhang Y</strong>, Syamala S, Seo JH, Bell C, O’Connor E, Liu Y, Schaeffer E, Karnes R, Weinmann S, Davicioni E, Morrissey C, Cejas P, Ellis L, Loda M, Wucherpfennig K, Pomerantz M, Spratt D, Corey E, Freedman M, Liu XS, Brown M, Long H#, Labbé D#. “<a href="https://www.biorxiv.org/content/10.1101/2021.04.23.441016v1" target="_blank">MYC drives aggressive prostate cancer by disrupting transcriptional pause release at androgen receptor targets.</a>” <strong><em>Nature Communications</em></strong>. (2022) Accepted. 
                        </li>
                        <br>

                        <li>
                            Zeng Z#, Wong C, Yang L, Ouardaoui N, Li D, Zhang W, Gu S, <strong>Zhang Y</strong>, Liu Y, Wang X, Fu J, Zhou L, Zhang B, Kim S, Yates K, Brown M, Freeman G, Uppaluri R, Manguso R, Liu XS#. “<a href="https://academic.oup.com/nar/article/50/D1/D1391/6371975" target="_blank">TISMO: syngeneic mouse tumor database to model tumor immunity and immunotherapy response.</a>”<strong><em> Nucleic Acids Research</em></strong>. (2022) 
                        </li>
                        <br>
                        <li>
                            Zhou L*, Zeng Z*, Egloff AM, Zhang F, Guo F, Campbell KM, Du P, Fu J, Zolkind P, Ma X, Zhang Z, <strong>Zhang Y</strong>, Wang X, Gu S, Riley R, Nakahori Y, Keegan J, Haddad R, Schoenfeld J, Griffith O, Manguso RT, Lederer JA, Liu XS#, Uppaluri R#. “<a href="doi:10.1136/jitc-2021-004034" target="_blank">Checkpoint blockade-induced CD8+ T cell differentiation in head and neck cancer responders.</a>” <strong><em>Journal for ImmunoTherapy of Cancer</em></strong>. (2022) 

                        </li>
                        <br>

                        <li>
                            Penter L, <strong>Zhang Y</strong>, Savell A, Huang T, Cieri N, Thrash EM, Kim-Schulze S, Jhaveri A, Fu J, Ranasinghe S, Li S, Zhang W, Hathaway E, Nazzaro M, Kim H, Chen H, Thurin M, Rodig S, Severgnini M, Cibulskis C, Gabriel S, Livak K, Cutler C, Antin J, Nikiforow S, Koreth J, Ho V, SArmand P, Ritz J, Steicher H, Neuberg D, Hodi F, Gnjatic S, Soiffer R, Liu SX, David M, Bachireddy P, Wu CJ#. “<a href="https://ashpublications.org/blood/article-abstract/137/23/3212/475524/Molecular-and-cellular-features-of-CTLA-4-blockade" target="_blank">Molecular and cellular features of CTLA-4 blockade for relapsed myeloid malignancies after transplantation.</a>” <strong><em>Blood</em></strong>. (2021) 
                        </li>
                        <br>

                        <li>
                            <strong>Zhang Y</strong>, Liu Y, Liu XS#. “<a href="https://www.nature.com/articles/s42256-021-00350-x" target='_blank'>Neural network architecture search with AMBER.</a>” <strong><em>Nature Machine Intelligence. News & Views</em></strong>. (2021) 3:372–3.
                        </li>
                        <br>

                        <li>
                            Gu SS*, Zhang W*, Wang X*, Jiang P*, Traugh N, Li Z, Meyer C, Stewig B, Xie Y, Bu X, Manos M, Front-Tello A, Gjini E, Lako A, Lim K, Conway J, Tewari A, Zeng Z, Sahu A, Tokleim C, Weirather J, Fu J, <strong>Zhang Y</strong>, Kroger B, Liang J, Cejas P, Freeman G, Rodig S, Long H, Gewurz B, Hodi F, Brown M#, Liu XS#. “<a href="https://cancerdiscovery.aacrjournals.org/content/11/6/1524" target="_blank">Therapeutically Increasing MHC-I Expression Potentiates Immune Checkpoint Blockade.</a>” <strong><em>Cancer Discovery</em></strong> (2021) 11:1524–41. 
                        </li>
                        <br>


                        <li>
                            Manjunath M*, Yan J*, Youn Y, Drucker KL, Kollmeyer TM, McKinney AM, Zazubovich V, <strong>Zhang Y</strong>, Costello J, Eckel-Passow J, Selvin P, Jenkins R, Song JS#. “<a href="https://academic.oup.com/neuro-oncology/article/23/4/638/5948532" target="_blank">Functional analysis of low-grade glioma genetic variants predicts key target genes and transcription factors.</a>” <strong><em>Neuro-Oncology</em></strong>. (2021) 23:638–49. 
                        </li>
                        <br>

                        <li>
                            Baur B, Schreiber J, Shin J, Zhang S, <strong>Zhang Y</strong>, Manjunath M, Song JS, Noble WS, Roy S#. “<a href="https://www.biorxiv.org/content/10.1101/(2021).08.29.458098v1"  target="_blank"> Leveraging epigenomes and three-dimensional genome organization for interpreting regulatory variation. </a>” Under Review. bioRxiv (2021).08.29.458098. 
                        </li>
                        <br>
<!--
                        <li>
                            Penter L, <strong><u>Y. Zhang</u></strong>, Savell A, Ranasinghe S, Huang T, Cieri N, et al. “<a href="https://doi.org/10.1182/blood-2020-139487" target="_blank">Local and Systemic Effects of Immune Checkpoint Blockade on Relapsed Myeloid Malignancies Following Allogeneic Hematopoietic Stem Cell Transplantation.</a>” <strong><em>Blood (ASH 2020)</em></strong>. (2020); 136:34–5.  
                        </li>
                        <br>
-->
                        <li>
                            Gu SS, Wang X, Hu X, Jiang P, Li Z, Traugh N, Bu X, Tang Q, Wang C, Zeng Z, Fu J, Meyer C, <strong>Zhang Y</strong>, Cejas P, Lim K, Wang J, Zhang W, Tokheim C, Sahu A, Xing X, Kroger B, Ouyang Z, Long H, Freeman G, Brown M, Liu XS. “<a href="21:263. https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02166-1" target="_blank">Clonal tracing reveals diverse patterns of response to immune checkpoint blockade.</a>” <strong><em>Genome Biology</em></strong>. (2020)  
                        </li>
                        <br>

                        <li>
                            Manjunath M, <strong>Zhang Y</strong>, Zhang S, Roy S, Perez-Pinera P, Song JS. “ABC-GWAS: Functional Annotation of Estrogen Receptor-Positive Breast Cancer Genetic Variants.” <strong><em>Frontiers in Genetics</em></strong>. (2020);0:730. 
                        </li>
                        <br>

                        <li>
						      
                            <strong>Zhang Y</strong>, Manjunath M, Yan J, Baur BA, Zhang S, Roy S, et al. “<a href="doi:10.3389/fgene.2019.00754" target="_blank">The cancer-associated genetic variant rs3903072 modulates immune cells in the tumor microenvironment.</a>” <strong><em>Frontiers in Genetics</em></strong> (2019); 
                        </li>
                        <br>

                        <li>
                            <strong>Zhang Y</strong>. “<a href="https://www.ideals.illinois.edu/handle/2142/105017" target="_blank">Functional interpretation of cancer-associated genetic variants.</a>” University of Illinois at Urbana-Champaign. (2019) (Dissertation)
                        </li>
                        <br>

                        <li>
                        <strong>Zhang Y</strong>, M. Manjunath*, S. Zhang, D. Chasman, S. Roy, and J.S. Song. "<a href="http://cancerres.aacrjournals.org/content/78/7/1579.long" target="_blank">Integrative genomic analysis predicts causative cis-regulatory mechanisms of the breast cancer-associated genetic variant rs4415084.</a>" <strong><em>Cancer Research</em></strong>, 78(7), 1579-1591, (2018). 
                        </li>
                        <br>

                        <li>
                            <strong>Zhang Y</strong>, Manjunath M, Kim Y, Heintz J, Song JS. "<a href="https://doi.org/10.1093/bioinformatics/bty794" target="_blank">SequencEnG: an Interactive Knowledge Base of Sequencing Techniques.</a>" <em><strong>Bioinformatics</strong></em>, bty794, (2019).<br>[<strong> <a href="http://education.knoweng.org/sequenceng/" target="_blank">SequencEnG page</a> </strong>]
                        </li>
                        <br>


                        <li>
                            Manjunath M, <strong>Zhang Y</strong>, Kim Y, Yeo SH, Sobh O, Russell N, et al. "<a href="https://peerj.com/articles/cs-155/" target="_blank" >ClusterEnG: an interactive educational web resource for clustering and visualizing high-dimensional data.</a>" <strong><em>PeerJ Computer Science</em></strong> 4, e155, (2018). [ <a href="http://education.knoweng.org/clustereng/" target="_blank">ClusterEnG page </a> ]
                        </li>

                        (*co-first authors, #corresponding author )

                        <br>
                        <br>
                        <br>
<!--
                    <a href="https://scholar.google.com/citations?user=yTJVsxEAAAAJ&hl=en" target="_blank"><strong><u> Google Scholar </u></strong></a>
-->
                    </ol>
                </div>

                <!-- <div class="col-2" > -->
                    <!-- altmetric sequenceng -->
                <!-- <div class='altmetric-embed' data-badge-type='donut' data-doi="10.1158/0008-5472.CAN-17-3486"></div> -->
                    <!-- altmetric cancerres -->
                 <!-- <div class='altmetric-embed' data-badge-type='donut' data-doi="doi.org/10.1101/319079"></div> -->
                    <!-- altmetric clustereng -->
                 <!-- <div class='altmetric-embed' data-badge-type='donut' data-doi="doi.org/10.7717/peerj-cs.155"></div> -->
                <!-- </div> -->


			</div>

        </section>

<!--		<section id="Poster Presentations">
			<div class="page-header">
				<h3>Poster Presentations</h3>
            </div>

			<div class="row">
                <div class="col-8">
                    <ol>
                        <li>
                        <strong><u>Y. Zhang</u></strong>, M. Manjunath, S. Zhang, D. Chasman, S. Roy, & J.S. Song. "Abstract 1220: Integrative genomic analysis discovers the causative regulatory mechanisms of a breast cancer-associated genetic variant." <em><strong>AACR Annual Meeting</strong>, 2018 Apr. 14-18, Chicago, IL</em>, (2018). 
                        </li>
                    </ol>
                    <br>
                    <ul>
                        <li>
                        <u>Y. Zhang</u>, M. Manjunath, & J.S. Song, "Identifying Causal Mechanisms of Germline Risk Variants in Breast Cancer."<em> Bioengineering Graduate Student Symposium, UIUC</em>, (2017). <br><strong>Best Poster Award.</strong>
                        </li>
                        <li>
                        <u>Y. Zhang</u>, P. Gad, & V.R. Edgerton, "Burst Detection in EMG of Stepping Spinal Cord Injured Rats." <em>UCLA Cross-disciplinary Scholars in Science and Technology (CSST) summer research No. 69, 2010 Sep., Los Angeles, CA, </em>(2013).
                        </li>
                    </ul>
                </div>

			</div>

            </section>
-->
    <a class="anchor" id="Projects-anchor"></a>
		<section id="Projects">
			<div class="page-header">
				<h3>Selected Projects</h3>
			</div>

        <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.101.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>1. Machine learning method on single-cell data Iintegration to understand tumor immunity</strong> [ Manuscript In Submission ]
           </div>
           </div>
           <br>


           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.001.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>2. Computational methods for interpretation of cancer-associated genetic variants.</strong> [ <a href="http://cancerres.aacrjournals.org/content/78/7/1579.long">Zhang et al. Cancer Research 2018</a> ]
               <p>Previous genome-wide association studies (GWAS) have identified common genetic variants that modulate cancer susceptibility, with their causative mechanisms missing. We have developed computational methods for functional interpretation of variants associated with cancer, by combining analyses of expression quantitative trait loci (eQTL), a modified version of allele-specific expression (ASE) that systematically utilizes haplotype information, transcription factor (TF) binding preference, and epigenetic information. Databases/techniques useful in this study are: TCGA, ENCODE, GTEx, Roadmap genomics, TF motif databases, genotyping array, RNA-seq, ChIP-seq, ChIA-PET, Hi-C, <em>etc.</em>. Our computational framework provides an effective means to integrate GWAS results with high-throughput genomic and epigenomic data. <br>
               </p>
           </div>
           </div>
           <br>

           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.002.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>3. Cancer-associated non-coding variant functional in tumor-infiltrating lymphocytes</strong> [ <a href="https://www.biorxiv.org/content/early/2018/12/13/493171">Zhang et al. Frontiers in Genetics 2019</a> ]
               <p>Functional interpretation of non-coding cancer variants has been successful, thanks to the epigenetic information in corresponding cancer cell types and matched normal tissues. However, this approach does not explore the potential effect of risk germline variants on other important cell types that constitute the microenvironment of tumor or its precursor. We show evidence that a breast cancer-associated variant may regulate a tumor-suppressing gene in tumor infiltrating lymphocytes, in particular, T lymphocytes and natural killer (NK) cells. Our hypothesis poses the possibility that cancer variants could be functional in immune cells in the tumor microenvironment, thereby modulating the immune surveillance and affecting the clearing of early cancer initiating cells. This was an idea flashed into my mind after a journal club in the Song Group. Thanks the immune cells for surveillance.<br>
               </p>
           </div>
           </div>
           <br>


           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.003.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>4. SequencEnG: an interactive knowledge base of NGS techniques.</strong> <br>[ <a href="https://www.biorxiv.org/content/early/2018/07/08/319079">Zhang et al. Bioinformatics 2019</a> ] &nbsp;  [ <a href="education.knoweng.org/sequenceng">SequencEnG page</a> ]&nbsp;
               <p>Next-generation sequencing (NGS) techniques are revolutionizing biomedical research by providing powerful methods for generating genomic and epigenomic profiles. However, a neat introductory learning resource is lacking. We have developed an interactive online educational resource called SequencEnG (acronym for <strong><u>Sequenc</u></strong>ing Techniques <strong><u>En</u></strong>gine for <strong><u>G</u></strong>enomics) to provide a tree-structured knowledge base of <strong>71</strong> different sequencing techniques and step-by-step NGS data analysis pipelines comparing popular tools. SequencEnG is part of the project KnowEnG (Knowledge Engine for Genomics). I wish I could have a resource like this when I first entered bioinformatics :D <br>
               Update 2018-08: SequencEnG has ~3500 users in 3 months!
               </p>
           </div>
           </div>
           <br>
<!--
           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.004.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>5. ClusterEnG: clustering and visualizing high-dimensional data.</strong><br> [ <a href="https://peerj.com/articles/cs-155/">Paper</a> ] &nbsp;  [ <a href="education.knoweng.org/clustereng">ClusterEnG page</a> ]&nbsp;
            <p>We have developed ClusterEnG (acronym for <u>Cluster</u>ing <u>En</u>gine for <u>G</u>enomics) to provide a web resource for online clustering analysis and interactive visualizations. ClusterEnG also serves as a barrier-free learning online resource of clustering algorithms with a focus on biomedical data. I have created cool visuliazations of clustering and dimensional reduction results for better understanding of data structure in an intuitive manner. 
            <br>
            </p>
           </div>
           </div>
           <br>

           <div class="row align-items-center">
           <div class='col-6 text-center'><img src="materials/project_pics/web_pics.005.png" width="600px" class="img-fluid" alt="Responsive image"></div>
           <div class='col-6'>
               <strong>6. An interactive resource for functional GWAS interpretation.</strong> <br>
               
               </p>
           </div>
           </div>
           <br>
-->
		</section>




    <a class="anchor" id="Courses-anchor"></a>
		<section id="Courses">
        <div class="page-header"><h3>Related Courses</h3></div>
        <div class="row">
        <div class="col-10">
            <table class="table table-bordered">
            <tbody>
            <tr>
                    <td>Mathematical Statistics</td>
                    <td>Machine Learning</td>
                    <td>Computational Cancer Biology</td>
                </tr>
            <tr>
                    <td>Stochastic Processes</td>
                    <td>Statistical Learning</td>
                    <td>Statistical Data Analysis in Physics</td>
            </tr>
            </tbody>
        </table>
        <br>
        <h5>Course Projects</h5>
        <em>Statistical Learning</em>: Prediction of Movie votes and campus shooting with SVM and gradient boosting models.<br>
        <em>Machine Learning</em>: Classification of NGS technique articles using natural language processing and SVM models.<br>
        <em>Stochastic Processes</em>: Segmentation of copy number using HMM based on genotype data in cancer cells.<br>
        <em>Statistical Data Analysis in Physics</em>: A cross-entropy method for graph clustering and Cheeger constant estimation.<br>

        <br>
        <h5> Workshop </h5>
            NHGRI Short Course on NGS: Technology & Statistical Methods, <em>Birmingham, AL</em>, 2016.<br>
            New England Future Faculty Workshop, Northeastern University, <em>Boston, MA </em>, 2020.
        
        <br>
        </div>
        </div>
        <br>
        </section>


<!--
    <section id="TEACHING">
      <div class="page-header"><h3>Talks & Teaching</h3></div>
            <ul>
                <li>Teaching Assistant, course materials <em>Harvard, High Impact Cancer Research Course</em>, 2020</li>
                <li>Invited Speaker, Bioinformatics Seminar, <em>UIUC ANSC590</em>, 2016, 2018</li>
                <li>Speaker, Bioengineering Graduate Student Seminar, <em>UIUC</em>, 2018</li>
                <li>Teaching Assistant, Biomedical Instrumentation Lab, <em>UIUC BIOE415</em>, 2014-2015</li>
            </ul>
    <br>
    </section>

 		<section id="AWARDS">
			<div class="page-header"><h3>Honors and Awards</h3></div>
      <ul>
        <table width="150%" align="center" border="0" cellpadding="20">
            <li><span class='title'>Best Poster Award, <em>Bioengineering Graduate Annual Symposium, UIUC</em>, 2017</li>
            <li><span class='title'>UCLA-CSST Summer Research Scholarship, <em>UCLA</em>, 2013</li>
            <li><span class='title'>Outstanding Undergraduate Student Honor, <em>USTC</em>, 2013</li>
            <li><span class='title'>Outstanding Student Scholarship, School of Life Sciences, <em>USTC</em>, 2013, 2014</li>
            <li><span class='title'>Aegon-Industrial Foundation Scholarship, <em>USTC</em>, 2012</li>
            <li><span class='title'>Panasonic Elite Scholarship, <em>USTC</em>, 2011</li>
         </table>
      </ul>
      </section> 





    <section id="PROF">
        <div class="page-header"><h3>Professional Experience</h3></div>
        <ul>
            <li>
            NHGRI Short Course on NGS: Technology & Statistical Methods, <em>Birmingham, AL</em>, 2016.
            </li>
            <li>
            BD2K (Big Data to Knowledge) KnowEnG center, <em>UIUC</em>, 2015-2018.
            </li>
        </ul>
    <br>
    </section>
    --> 
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>

      <!-- Footer ================================================== -->
            <footer class="page-footer font-small">
                <!-- Copyright -->
                <div class="footer-copyright text-center py-3" style="background-color:#FFFFFF">
                <font color='green'>© 2022 Copyright: <strong> Yi Zhang</strong> (updated May 2022)</font>
                </div>
                </footer>

</body></html>
